sdk/metricsadvisor/azure-ai-metricsadvisor/src/main/java/com/azure/ai/metricsadvisor/implementation/models/HookInfo.java

```diff
@@ -132,4 +132,15 @@ public HookInfo setExternalLink(String externalLink) {
     public List<String> getAdmins() {
         return this.admins;
     }
+
+    /**
+     * Set the admins property: hook administrators.
```

Comment:
Need to update the javadoc to make it more clear.

Reply:
This is an auto-gen file, but we have a better documentation in the hand-written type wrapping this file (NotificationHook) like below:

> The list of user e-mails with administrative rights to manage the hook.

Reply:
Oh its implementation file, please ignore.